pkg/services/sqlstore/quota.go

```diff
@@ -78,17 +84,21 @@ func GetOrgQuotas(query *models.GetOrgQuotasQuery) error {
 
 	result := make([]*models.OrgQuotaDTO, len(quotas))
 	for i, q := range quotas {
-		// get quota used.
-		rawSQL := fmt.Sprintf("SELECT COUNT(*) as count from %s where org_id=?", dialect.Quote(q.Target))
-		resp := make([]*targetCount, 0)
-		if err := x.SQL(rawSQL, q.OrgId).Find(&resp); err != nil {
-			return err
+		var used int64
+		if q.Target != "alert_rule" || query.IsNgAlertEnabled {
```

Comment:
Looks like you're missing the use of the `ALERT_RULE_TARGET` const here and also line 199 😄